config/locales/en.yml

```diff
@@ -42,7 +45,9 @@ en:
     url_funding_and_support: https://getintoteaching.education.gov.uk/funding-and-support
     url_get_an_adviser_signup: https://getintoteaching.education.gov.uk/teacher-training-adviser/sign_up/identity
     url_get_an_adviser_start: https://getintoteaching.education.gov.uk/teacher-training-advisers
+    url_if_your_application_is_unsuccessful: https://getintoteaching.education.gov.uk/how-to-apply-for-teacher-training/if-your-application-is-unsuccessful
```

Comment:
Are we not following this pattern for GIT links anymore? 

https://github.com/DFE-Digital/apply-for-teacher-training/blob/9cb14ad2db2d06a89b70cee5ce6982c0932a26aa/config/locales/en.yml#L35

Reply:
I'm not sure what you mean. What should it be instead?

Reply:
All of our get into teaching links are in en.yml under `get_into_teaching` and are prefixed with `url`, so it would be good to keep to this pattern so they are all in the same place.

Reply:
This is under `get_into_teaching` and it start with `url`. Am I missing something?

Reply:
No worries, It's a big file, easy to get a bit lost! Thanks for the very thorough and thoughtful review!